integration/spark/src/main/java/openlineage/spark/agent/lifecycle/plan/PlanUtils.java

```diff
@@ -227,4 +234,16 @@ static Path getDirectoryPath(Path p, Configuration hadoopConf) {
       return p;
     }
   }
+
+  public static final Function<String, Optional<UUID>> convertToUUID = tryConvert(UUID::fromString);
```

Comment:
This is never used as a `Function` reference. Just make it a static method. You can always use a method reference as a `Function` if you need to.